dev/puppet-server.conf.sample

```diff
@@ -30,8 +30,8 @@ webserver: {
 # configure the mount points for the web apps
 web-router-service: {
     "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
-    "puppetlabs.services.master.master-service/master-service": "/puppet"
-
+    "puppetlabs.services.master.master-service/master-service": {master-routes: "/puppet"
+                                                                 404:           "/" }
```

Comment:
That's a good idea, I'll change the name. This response will only be returned for requests that are missing the configurable `puppet` and `puppet-ca` prefixes.